hawkular-inventory-api/src/main/java/org/hawkular/inventory/base/BaseTenants.java

```diff
@@ -46,7 +46,7 @@ private BaseTenants() {
 
     }
 
-    public static class ReadWrite<BE> extends Mutator<BE, Tenant, Tenant.Blueprint, Tenant.Update>
+    public static class ReadWrite<BE> extends Mutator<BE, Tenant, Tenant.Blueprint, Tenant.Update, String>
```

Comment:
That's the actual `java.lang.String`. `Mutator` has an `Id` type parameter and `ReadWrite` extends mutator with the `Id` type param "realized" with `String`.